docs/topics/wasm/wasm-get-started.md

```diff
@@ -1,117 +1,112 @@
+   * **Name:** WasmDemo
+   * **Group:** wasm.project.demo
+   * **Artifact:** wasmdemo
 
-1. In IntelliJ IDEA, open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
-   
-   You can find the Gradle tasks in the Gradle tool window once the project loads.
-
-   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
+   > If you use the web wizard, specify "WasmDemo" as Project Name and "wasm.project.demo" as Project ID.
    >
    {style="note"}
 
-2. In **wasmdemo** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDevelopmentRun** task.
+5. Select the **Web** target. Make sure that no other options are selected.
+6. Click **Create**.
 
-   ![Run the Gradle task](wasm-gradle-task-window.png){width=400}
+   ![Kotlin Multiplatform wizard](wasm-kmp-wizard.png){width=600}
 
-   Alternatively, you can run the following command in the terminal from the `WasmDemo` root directory:
+## Run the application
 
-   ```bash
-   ./gradlew wasmJsBrowserDevelopmentRun -t
-   ```
+Once the project loads, select **composeApp [wasmJs]** in the list of run configurations and click **Run**.
 
-3. Once the application starts, open the following URL in your browser:
+![Run the Compose Multiplatform app on web](compose-run-web-black.png){width=300}
 
-   ```bash
-   http://localhost:8080/
-   ```
+The web application opens automatically in your browser. Alternatively, you can type the following URL in your browser when the run is finished:
 
-   > The port number can vary because the 8080 port may be unavailable. You can find the actual port number printed
-   > in the Gradle build console.
-   >
-   {style="tip"}
+```shell
+   http://localhost:8080/
+```
+> The port number can vary because the 8080 port may be unavailable.
+> You can find the actual port number in the Gradle build console.
+>
+{style="tip"}
 
-   You see a "Click me!" button. Click it:
+You see a "Click me!" button. Click it:
 
-   ![Click me](wasm-composeapp-browser-clickme.png){width=650}
+![Click me](wasm-composeapp-browser-clickme.png){width=600}
 
-   Now you see the Compose Multiplatform logo:
+Now you see the Compose Multiplatform logo:
 
-   ![Compose app in browser](wasm-composeapp-browser.png){width=650}
+![Compose app in browser](wasm-composeapp-browser.png){width=600}
 
 ## Generate artifacts
 
-In **wasmdemo** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDistribution** task.
+Generate your project's artifacts to publish on a site:
 
-![Run the Gradle task](wasm-gradle-task-window-compose.png){width=400}
+1. Open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+2. In **wasmdemo** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDistribution** task.
 
-Alternatively, you can run the following command in the terminal from the `WasmDemo` root directory:
+   > You need at least Java 11 as your Gradle JVM for the tasks to load successfully.
```

Comment:
I think we recommend at least Java 17 for Compose Multiplatform, let's mention it here? So that it doesn't look that Java 11 is a satisfactory baseline for the whole thing. "You need at least Java 11 <...>, and we recommend at least Java 17 for Compose Multiplatform projects in general."